src/Paket.VisualStudio/PaketErrorPane.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Shell.Interop;
+using System;
+using Microsoft.VisualStudio.Shell;
+using EnvDTE;
+
+namespace Paket.VisualStudio
+{
+    internal class PaketErrorPane
+    {
+        private static ErrorListProvider vsbaseWarningProvider;
```

Comment:
If I press F1 on the error it should open a webpage, right?

2015-05-12 20:35 GMT+02:00 Sam Harwell <notifications@github.com>:

> In src/Paket.VisualStudio/PaketErrorPane.cs
> <https://github.com/hmemcpy/Paket.VisualStudio/pull/21#discussion_r30165866>
> :
>
> > @@ -0,0 +1,60 @@
> > +using System.Collections.Generic;
> > +using Microsoft.VisualStudio.Text;
> > +using Microsoft.VisualStudio.Shell.Interop;
> > +using System;
> > +using Microsoft.VisualStudio.Shell;
> > +using EnvDTE;
> > +
> > +namespace Paket.VisualStudio
> > +{
> > +    internal class PaketErrorPane
> > +    {
> > +        private static ErrorListProvider vsbaseWarningProvider;
>
> @forki <https://github.com/forki> I'm not sure what you mean.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hmemcpy/Paket.VisualStudio/pull/21/files#r30165866>.
>